src/test/java/cms/gov/madie/measure/services/ElmTranslatorClientTest.java

```diff
@@ -46,6 +52,8 @@ void beforeEach() {
         .thenReturn("/cql/translator/cql");
   }
 
+  private final String HEADER_1 = "api-key";
```

Comment:
lets move this up in variable section. seeing some codacy violations

Reply:
@adongare I moved it up, can you please help review again? Thanks so much!!